plugins/measure/src/measure.hh

```diff
@@ -0,0 +1,79 @@
+/*
+ * Scroom - Generic viewer for 2D data
+ * Copyright (C) 2009-2020 Kees-Jan Dijkzeul
+ *
+ * SPDX-License-Identifier: LGPL-2.1
+ */
+
+#pragma once
+
+#include <scroom/plugininformationinterface.hh>
+#include <scroom/utilities.hh>
+#include <scroom/viewinterface.hh>
+
+class MeasureHandler : public PostRenderer, public SelectionListener, virtual public Scroom::Utils::Base
+{
+public:
+  MeasureHandler();
+
+public:
+  typedef boost::shared_ptr<MeasureHandler> Ptr;
+
+private:
+  Selection* selection;
+
+public:
+  ViewInterface::Ptr view;
```

Comment:
You cannot store a `ViewInterface` pointer in anything that is kept alive by a `View`.

I would prefer that you pass in a `ViewInterface::Ptr` on methods where it is needed. Alternatively, store a `weak_ptr`

Reply:
Figured that might have been the case, I'll add them to the methods since implementations will likely need the view for something either way.